core/src/store/function.rs

```diff
@@ -0,0 +1,32 @@
+use crate::data::CustomFunction;
+use crate::result::{Error, Result};
+use async_trait::async_trait;
```

Comment:
nit. this is the project specific code convention.
we prefer bundling `use` statement into a single one. it is for letting `cargo fmt` to format all by itself.

then here
```rust
use {
    crate::{
        data::CustomFunction,
        result::{Error, Result},
    },
    async_trait::async_trait,
};
```